movement/io/load_poses.py

```diff
@@ -0,0 +1,58 @@
+import logging
+from pathlib import Path
+from typing import Optional, Union
+
+import pandas as pd
+
+from movement.io.validators import DeeplabcutPosesFile
+from movement.log_config import configure_logging
+
+# initialize logger
+configure_logging()
+logger = logging.getLogger(__name__)
+
+# TODO:
```

Comment:
Very pedantic, but I find `TODO`'s end up getting forgotten. This is fairly key, so it probably won't, but I always prefer a GitHub issue for discussion, tracking etc.

Reply:
Good point. I will delete from here, since it's already an issue.